p2p-specs/p2p-interface.md

```diff
@@ -606,12 +617,10 @@ class UserOp(Container):
     signature: bytes
 ```
 
-#### `UserOperationsWithEntryPoint`
+#### `VerifiedUserOperation`
 
 ```python
-class UserOperationsWithEntryPoint(Container):
-    entry_point_contract: Address
+class VerifiedUserOperation(Container):
+    user_operation: UserOp
     verified_at_block_hash: uint256
```

Comment:
as an optimization, we don't need to send entire hash: the message is completely ignored if the receiver doesn't know this recent hash, so it is enough to send `uint32`
(same goes for entrypoint, which I think we should keep here..)

Reply:
Adding the entrypoint back, I think we should keep the full block hash for easy lookups, if we really need to save bandwidth we can probably optimize elsewhere.